pkg/apis/build/v1alpha1/buildrun_types.go

```diff
@@ -253,15 +258,15 @@ type Condition struct {
 
 	// LastTransitionTime last time the condition transit from one status to another.
 	// +optional
-	LastTransitionTime metav1.Time `json:"lastTransitionTime,omitempty" description:"last time the condition transit from one status to another"`
+	LastTransitionTime *metav1.Time `json:"lastTransitionTime,omitempty" description:"last time the condition transit from one status to another"`
```

Comment:
Optional string (time is like a string with a specific format) where we are not required to distinguish `nil` and `""`. See discussion above.